From d23ecd8c3055f492bb6ff88b28e7cbeb1866d4eb Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Tue, 5 Sep 2017 17:22:20 +0000
Subject: [PATCH 261/462] Mantis 0046781: IPS Admin application

added JarAuthenticationClient and JarSoapAuthenticationClient for authenicationWClient
added JarAdminApplication for admin Application



git-svn-id: svn://localhost/SelfServiceBuild/trunk@418 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/java/AntProperties.xml | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/SelfServiceBuild/Build/java/AntProperties.xml b/SelfServiceBuild/Build/java/AntProperties.xml
index e8c4c01..e253c43 100644
--- a/SelfServiceBuild/Build/java/AntProperties.xml
+++ b/SelfServiceBuild/Build/java/AntProperties.xml
@@ -195,7 +195,7 @@
 <property name="JarJunit"                   value="${DIR_GLOBALS}/ThirdParty/junit/4.10/junit-4.10.jar"/>
 
 <!-- jMockIt -->
-<property name="JarJMockIt"                   value="${DIR_GLOBALS}/ThirdParty/jmockit/1.8/jmockit.jar"/>
+<property name="JarJMockIt"                 value="${DIR_GLOBALS}/ThirdParty/jmockit/1.8/jmockit.jar"/>
 
 <!-- scxml -->
 <property name="JarSCXML"                   value="${DIR_GLOBALS}/ThirdParty/commons-scxml/0.9/commons-scxml-0.9.jar"/>
@@ -210,6 +210,10 @@
 <!-- BarcodeParser -->
 <property name="JarBarcodeParser"           value="${COMMONROOT}/Massai/java/barcodeParserLib/target/barcodeParserLib-1.2.0.1.jar"/>
 
+<!-- BarcodeParser -->
+<property name="JarAuthenticationClient"    value="${DIR_GLOBALS}/ThirdParty/authenticationWClient/1.1-SNAPSHOT/authenticationWClient-1.1-SNAPSHOT.jar"/>
+<property name="JarSoapAuthenticationClient" value="${DIR_GLOBALS}/ThirdParty/authenticationWClient/1.1-SNAPSHOT/soapAuthenticationClient-1.1-SNAPSHOT.jar"/>
+
 <!-- JRE -->
 <property name="JarAppletPlugin"            value="${JAVA_HOME}/jre/lib/plugin.jar"/>
 
@@ -245,6 +249,7 @@
 <property name="JarMatacClient"             value="${JarDir}/MATAC-CLIENT.jar"/>
 <property name="JarMatac"                   value="${JarDir}/MATAC-XBEANS.jar"/>
 <property name="JarApplicationAdapter"      value="${JarDir}/ApplicationAdapter.jar"/>
+<property name="JarAdminApplication"        value="${JarDir}/AdminApplication.jar"/>
 <property name="JarMassaiEventReporting"    value="${JarDir}/massai-eventreporting.jar"/>
 <property name="JarLoggingClient"           value="${JarDir}/LoggingClient.jar"/>
 <property name="JarLoggingInterface"        value="${JarDir}/LoggingInterface.jar"/>
-- 
2.41.0.windows.1

